README.md

```diff
@@ -24,89 +40,67 @@ Or if you want to use the latest Feathr code from GitHub:
 pip install git+https://github.com/linkedin/feathr.git#subdirectory=feathr_project
 ```
 
-## Quick Start
+## Running Feathr Examples
```

Comment:
Running Feathr Examples should be at the bottom as it's too many tutoirals examples already with the azure one-click thing.